frontend/src/components/atoms/ProjectionRowEntry.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+
+type ITransaction = import('../../declarations/transaction').ITransaction;
```

Comment:
Great use of type imports!

Reply:
Thanks, I looked at how it was done elsewhere and copied that :D